KarhooUISDK/Screens/CheckoutScreen/NewCheckout/NewCheckoutCoordinator.swift

```diff
@@ -121,6 +124,31 @@ extension KarhooNewCheckoutCoordinator: NewCheckoutRouter {
         viewController.modalPresentationStyle = .overFullScreen
         baseViewController.present(viewController, animated: true, completion: nil)
     }
+    
+    func routeToComment(title: String, comments: String) {
+        guard let presenter = presenter else { return }
+        let bottomSheetViewModel = KarhooBottomSheetViewModel(
+            title: title) { [weak self] in
```

Comment:
You need to decide between a one-liner or muli-line call.

Reply:
@BartlomiejSopalaKarhoo I think you missed this \n between `title` and `)`